### DISCUSSION TITLE
Weird bug on mobile. Should I open an issue? Ref: D580:1A63:8FA2:13D95
:6311C2CC

### DISCUSSION THREAD

**Discussion by killshot13** [community/community#31747] [Mobile] [answered]
Was browsing through the `Explore` feed this morning and noticed a strange thing that happens on the mobile app. 

If I select the "profile" of GitHub Actions, (on desktop this just links to [the GH Actions landing page](https://github.com/features/actions)) made available when the aforementioned receives credit for a publishing a given version release, etc., instead of the redirect I get a strange error.

---

>Something went wrong while
executing your query. Please
include `D580:1A63:8FA2:13D95
:6311C2CC` when reporting this
issue.

---

I'm thinking this is just because there is no default profile matching `github-actions[bot]`? Anyone have any thoughts or inputs on this topic? 🤔 

---

![Screenshot_20220902-071834_GitHub.jpg](https://user-images.githubusercontent.com/47695475/188143084-48930b01-f663-4322-b6ba-0d2752ce025c.jpg)

![Screenshot_20220902-034616_GitHub.jpg](https://user-images.githubusercontent.com/47695475/188143048-a80e022a-ce17-4957-8efe-aafad8cf3773.jpg)

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @killshot13! You are totally correct in your assumption, we try to open the profile of a bot and that fails to load because bots don't have profiles. I'll create an internal issue to track it, and I hope we have a fix ready soon. 

Update: Could you share in which platform you are experiencing this issue?

**Reply by killshot13** [reply]
Sure thing!

I observed this error while using the GitHub mobile application (v1. 78.0) on a Samsung mobile phone running the Android 12 OS.
